isis/src/base/objs/ImagePolygon/ImagePolygon.cpp

```diff
@@ -1329,6 +1330,11 @@ namespace Isis {
       msg += "isis operation exception [" + IString(e.what()) + "]";
       throw IException(e, IException::Unknown, msg, _FILEINFO_);
     }
+    catch(std::exception &e) {
+      std::string msg = "Caught std::exception: ";
+      msg += e.what();
+      throw IException(IException::Unknown, msg, _FILEINFO_); 
+    }
```

Comment:
std::exception wasn't being caught causing the real error of an incomplete linestring to be swllowed by `catch(...)`